packages/common/package.json

```diff
@@ -48,13 +47,11 @@
     "@types/react": "^16.4.14",
     "@types/react-dom": "^16.0.7",
     "@types/styled-components": "^4.1.0",
-    "babel-loader": "^8.0.4",
```

Comment:
So we're completely removing babel-loader?

Reply:
I don't believe we have a direct dependency anymore; this was used by Storybook previously IIRC.